### PR TITLE
V3.11

### PR DESCRIPTION
**Pull request by MudasarUKCloud**

### CODE REVIEW

File: jenkins-pipelines/openshift/Jenkinsfile [hunk 1]
```diff
@@ -75,6 +78,10 @@ node ('openstack-jenkins-slave') {
                 echo "  node_routes: \$NODE_ROUTES"                   >> environment.yaml
                 echo "  multinetwork: \$MULTINETWORK"                 >> environment.yaml
                 echo "  deploy_extra_gateway: \$DEPLOYEXTRAGATEWAY"   >> environment.yaml
+                echo "  registry_details:"                            >> environment.yaml 
+                echo "  registry_url: \$REG_URL"                      >> environment.yaml
```
Comment (benbacon) [inline jenkins-pipelines/openshift/Jenkinsfile]:
```suggestion
                echo "    registry_url: \$REG_URL"                      >> environment.yaml
```

File: jenkins-pipelines/openshift/Jenkinsfile [hunk 2]
```diff
@@ -75,6 +78,10 @@ node ('openstack-jenkins-slave') {
...
+                echo "  registry_user: \$REG_USER"                    >> environment.yaml
```
Comment (benbacon) [inline jenkins-pipelines/openshift/Jenkinsfile]:
```suggestion
                echo "    registry_user: \$REG_USER"                    >> environment.yaml
```

File: jenkins-pipelines/openshift/Jenkinsfile [hunk 3]
```diff
@@ -75,6 +78,10 @@ node ('openstack-jenkins-slave') {
...
+                echo "  registry_password: \$REG_PASSWORD"            >> environment.yaml
```
Comment (benbacon) [inline jenkins-pipelines/openshift/Jenkinsfile]:
```suggestion
                echo "    registry_password: \$REG_PASSWORD"            >> environment.yaml
```

**Review by benbacon — COMMENTED** [review]
Looks good minus missing indentation on registry_details key

**Review by benbacon — APPROVED** [review]
Looks good

**Comment by benbacon** [comment]
@stevemul did we still want a separate branch for 3.10? If so we shouldn't be merging this into master right and it should remain a separate branch

**Comment by gellner** [comment]
No longer required - all commits were included on another PR

**Pull request closed without merging**